evap/evaluation/templates/textanswer_visibility_info.html

```diff
@@ -1,7 +1,7 @@
 <span class="far fa-eye ml-1 icon-gray" data-toggle="tooltip" data-placement="left" title="
     {{ intro_text }}
     {% for contributor in visible_by_contribution %}
-        {{ contributor.full_name }}{% if not forloop.last or visible_by_delegation_count > 0 %}, {% endif %}
+        {{ contributor.full_name }}{% if contributor.is_proxy_user %}{% blocktrans count delegate_count=contributor.delegates.count %} ( {{ delegate_count }} person ){% plural %} ( {{ delegate_count }} persons ){% endblocktrans %}{% endif %}{% if not forloop.last or visible_by_delegation_count > 0 %}, {% endif %}
```

Comment:
@janno42 says this is not one of the one correct versions to whitespace this (should be `(n persons)` without inner spaces)